### PR TITLE
Update Quickstart for v0.8

### PR DESCRIPTION
**Pull request by houhuiting**
Fixes: https://github.com/confidential-containers/enclave-cc/issues/181
- Add the content of deploy KBS cluster and create encrypted image in enclave-cc.md
- Delete verdictd in enclave-cc.md and add cc-kbc and sample-kbc content, and give examples of usage
- Modify the creation of enclave-cc custom resource in quickstart.md

### CODE REVIEW

File: guides/enclave-cc.md [hunk 1]
```diff
@@ -7,27 +7,48 @@ section of the [quickstart guide](../quickstart.md).
...
 
+**Note** Before configuring KBC, please refer to the [guide](../quickstart.md#deploy-and-configure-tenant-side-coco-key-broker-system-cluster) to deploy KBS cluster.
+
 **Note** The KBC configuration changes to the enclave-cc custom resource yaml 
 must be made **before** deploying it. 
 
-Enclave CC supports Verdictd and in order to use it, users will have to
-properly configure a decrypt_config.conf, in order to set the `KBC` (`sample_kbc`
-or `eaa_kbc`) `IP`,`PORT`, and the `SECURITY_VALIDATE` (`false` or  `true`)
+Enclave CC supports cc-kbc and sample-kbc, in order to use it, users will have to
```
Comment (Xynnn007) [inline guides/enclave-cc.md:15]:
```suggestion
Enclave CC supports cc-kbc and sample-kbc, in order to use them, users will have to
```

File: guides/enclave-cc.md [hunk 2]
```diff
@@ -7,27 +7,48 @@ section of the [quickstart guide](../quickstart.md).
...
+or `sample_kbc`) `IP`,`PORT`, and the `SECURITY_VALIDATE` (`false` or  `true`).
+
 ```json
 {
     "key_provider": "provider:attestation-agent:KBC::IP:PORT",
      "security_validate": SECURITY_VALIDATE
 }
 ```
 
+The following is an example of cc_kbc:
```
Comment (Xynnn007) [inline guides/enclave-cc.md:26]:
```suggestion
The following is an example of `cc_kbc`:
```

File: guides/enclave-cc.md [hunk 3]
```diff
@@ -7,27 +7,48 @@ section of the [quickstart guide](../quickstart.md).
...
+
+```json
+{
+    "key_provider": "provider:attestation-agent:cc_kbc::http://127.0.0.1:8080",
+    "security_validate": true
+}
+```
+The following is an example of sample_kbc:
```
Comment (Xynnn007) [inline guides/enclave-cc.md:34]:
```suggestion
The following is an example of `sample_kbc`:
```

File: guides/enclave-cc.md [hunk 4]
```diff
@@ -7,27 +7,48 @@ section of the [quickstart guide](../quickstart.md).
...
+
+```json
+{
+    "key_provider": "provider:attestation-agent:sample_kbc::127.0.0.1:50000",
+    "security_validate": false
```
Comment (Xynnn007) [inline guides/enclave-cc.md:39]: Why is this `false` while `cc_kbc`'s true

**Review by Xynnn007 — APPROVED** [review]
Thanks for the contribution and welcome to Confidential Containers! @houhuiting

File: guides/enclave-cc.md (hunk 4 above)
Comment (mythi) [reply guides/enclave-cc.md:39]: I never looked to enable it since I had missed the `get_resource`  addition to it. let's add a TODO item?

Comment (Xynnn007) [reply guides/enclave-cc.md:39]: ok. let s do it then

File: guides/enclave-cc.md [hunk 5]
```diff
@@ -7,27 +7,48 @@ section of the [quickstart guide](../quickstart.md).
...
+
 Once that's set according to the users needs, the user will then have to run:
 `cat decrypt_config.conf | base64 -w 0` in order to get the data encoded and
 set it accordingly [here](https://github.com/confidential-containers/operator/blob/6f241fbc056f0a5d9e1bd2c10b2cedc0782b99ff/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml#L124).
 
 ## Creating a sample CoCo workload using enclave-cc
 
 As an example, we setup a sample *hello world*
-workload with an encrypted and cosign signed container image using the `enclave-cc` runtime class for process based TEEs.
+workload with an encrypted and cosign signed container image using the `enclave-cc` runtime class for process based TEEs. This encrypted image is only used for testing. If you want to use it in your own production use cases, please refer to the [guide](../quickstart.md#encrypting-an-image) to create a new encrypted image and deploy it.
```
Comment (dcmiddle) [inline guides/enclave-cc.md:50]: Please format long lines at 100 characters or at sentence breaks. This makes diffs clearer later.

**Comment by mythi** [comment]
@houhuiting Could you also edit the first comment entry in this PR to add `Fixes: https://github.com/confidential-containers/enclave-cc/issues/181` (with a link to our issue)

**Comment by houhuiting** [comment]
> @houhuiting Could you also edit the first comment entry in this PR to add `Fixes: https://github.com/confidential-containers/enclave-cc/issues/181` (with a link to our issue)

Of course, it has been modified now

File: guides/enclave-cc.md (hunk 5 above)
Comment (houhuiting) [reply guides/enclave-cc.md]: it has been modified now.

**Review by dcmiddle — APPROVED** [review]

**Review by fitzthum — APPROVED** [review]
LGTM

**Pull request merged**